package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "blockchain-wallet-v4",
-  "version": "4.78.4",
+  "version": "4.77.2-recovery-test",
```

Comment:
this will probably conflict when you try to merge, you may just want to undo this change unless you def need it.